test-utils-core/pom.xml

```diff
@@ -84,6 +84,11 @@
             <groupId>junit</groupId>
             <artifactId>junit</artifactId>
         </dependency>
+        <dependency>
+            <groupId>com.googlecode.java-diff-utils</groupId>
+            <artifactId>diffutils</artifactId>
+            <version>1.3.0</version>
```

Comment:
Please manage the dependency version in a dependencyManagement tag in the parent pom. it doesn't need to go in the common bom, but we try to avoid putting version numbers in our child poms.

Reply:
done for the next commit